File: level1/5 - Tests et conditions/7 - Concours de tir à la corde.c
//coded by cybereagle2001
#include <stdio.h>
#define repeat(nb) for(int _loop = 1, _max = (nb); _loop <= _max; _loop++)

int main()
{
   int nbPersonnes;
   scanf("%d", &nbPersonnes);
   int totalEquipe1 = 0, totalEquipe2 = 0;
   repeat (nbPersonnes)
   {
      int poids1, poids2;
      scanf("%d%d", &poids1, &poids2);
      totalEquipe1 = totalEquipe1 + poids1;
      totalEquipe2 = totalEquipe2 + poids2;
   }
   if (totalEquipe1 > totalEquipe2)
   {
      printf("L'équipe 1 a un avantage\n");
   }
   else
   {
      printf("L'équipe 2 a un avantage\n");
   }
   printf("Poids total pour l'équipe 1 : %d\n", totalEquipe1); 
   printf("Poids total pour l'équipe 2 : %d\n", totalEquipe2);
}

File: java/level 1/5- Tests et conditions/7- Concours de tir à la corde.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input = new Scanner(System.in);
   public static void main(String[] args){
      int  nbMembres = input.nextInt();
      int team1=0;
      int team2=0;
      int j=0; int y=0;
      for (int loop=0; loop<nbMembres; loop++){
         int poids= input.nextInt();
         int poids2= input.nextInt();
         j=j+poids;
         y=y+poids2;
      }
      if (j>y){
         System.out.println("L'équipe 1 a un avantage");
      }
      else if(y>j){
         System.out.println("L'équipe 2 a un avantage");
      }
      System.out.println("Poids total pour l'équipe 1 : "+j);
      System.out.println("Poids total pour l'équipe 2 : "+y);
   }
}
